src/TitleForm/TitleFormContainer.js

```diff
@@ -0,0 +1,71 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import ReactRouterPropTypes from 'react-router-prop-types';
+
+import { stripesConnect } from '@folio/stripes/core';
+
+import {
+  identifierTypesResource,
+  titlesResource,
+} from '../common/resources';
+import TitleForm from './TitleForm';
+
+function TitleFormContainer({ history, match, mutator }) {
+  const titleId = match.params.id;
+  const [identifierTypes, setIdentifierTypes] = useState();
+
+  useEffect(() => {
+    mutator.identifierTypes.GET()
+      .then(setIdentifierTypes)
+      .catch(() => setIdentifierTypes([]));
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [titleId]);
+
+  const onCancel = useCallback(
+    () => history.push('/receiving'),
+    [],
+  );
+  const onSubmit = useCallback(
+    ({ poLineNumber, ...newTitle }) => {
+      return mutator.titles.POST(newTitle)
+        .then(savedTitle => {
+          setTimeout(() => history.push('/receiving'));
+        });
+    },
+    [history, titleId],
+  );
+
+  if (!identifierTypes) {
+    return null;
```

Comment:
Why not `<LoadingPane>`?

Reply:
good question, null is simpler to use in any component, when LoadingPane return a Pane that requires a Paneset above in hierarchy